mappings/net/minecraft/client/render/FirstPersonRenderer.mapping

```diff
@@ -0,0 +1,51 @@
+CLASS none/box net/minecraft/client/render/FirstPersonRenderer
+	FIELD a TEXTURE_MAP_BACKGROUND Lnone/kp;
+	FIELD b TEXTURE_UNDERWATER Lnone/kp;
+	FIELD c minecraft Lnone/bdr;
+	FIELD d mainhand Lnone/aes;
+	FIELD e offhand Lnone/aes;
+	FIELD f equipProgressMainhand F
+	FIELD g previousEquipProgressMainhand F
+	FIELD h equipProgressOffhand F
+	FIELD i previousEquipProgressOffhand F
+	FIELD j renderManager Lnone/btd;
+	FIELD k itemRenderer Lnone/btr;
+	METHOD <init> (Lnone/bdr;)V
+		ARG 0 minecraft
+	METHOD a updateHeldItems ()V
+	METHOD a renderFirstPersonItem (F)V
+		ARG 0 deltaTicks
+	METHOD a rotate (FF)V
+		ARG 0 degreeX
+		ARG 1 degreeY
```

Comment:
`pitch` / `yaw` ? Unsure what this actually is and which is which if it would apply.